content_delegate.js

```diff
@@ -240,11 +240,6 @@ ContentDelegate.prototype.onDocumentViewSubmit = function (event) {
     let image_string = $('td:contains(Image)').text();
     let regex = /(\d+)-(\d+)/;
     let matches = regex.exec(image_string);
-    if (!matches) {
```

Comment:
Removing this safety check is a mistake.
We should always have a plan if the regexp fails, e.g. the case number lacks a hyphen, the receipt structure is different, or there's a letter prefix in the docket number `10A-22`, or who knows.

I'd say, then, that the function should return after calling `form.submit()` rather than continuing to execute. And we know this code path is unlikely to be well-utilized, but it is a safety bound and that is a wise thing.